operator_ui/src/pages/Keys/Index.test.tsx

```diff
@@ -1,60 +0,0 @@
-import React from 'react'
```

Comment:
Deleted this test because of issues with using Apollo client and the REST API. 

A new test will be written once all keys have been converted to use the GQL API